packages/wrangler/src/__tests__/secret.test.ts

```diff
@@ -95,6 +96,83 @@ describe("wrangler secret", () => {
 				expect(std.err).toMatchInlineSnapshot(`""`);
 			});
 
+			it("should create secret:bulk", async () => {
+				writeFileSync(
+					"secret.json",
+					JSON.stringify({
+						"secret-name-1": "secret_text",
+						"secret-name-2": "secret_text",
+					})
+				);
+
+				// User counter to pass different secrets to the request mock
+				const counter = 0;
+				setMockResponse(
+					`/accounts/:accountId/workers/scripts/:scriptName/secrets`,
+					"PUT",
+					([_url, accountId]) => {
+						expect(accountId).toEqual("some-account-id");
+						counter + 1;
+
+						return { name: `secret-name-${counter}`, type: "secret_text" };
+					}
+				);
+
+				await runWrangler("secret:bulk ./secret.json --name script-name");
+
+				expect(std.out).toMatchInlineSnapshot(`
+			"🌀 Creating the secrets for the Worker \\"script-name\\"
+			✨ Successfully created secret for key: secret-name-1
+			✨ Successfully created secret for key: secret-name-2
+			✨ Successfully created secrets"
+		`);
+				expect(std.err).toMatchInlineSnapshot(`""`);
+			});
+
+			it("should handle failed secret:bulk", async () => {
+				writeFileSync(
+					"secret.json",
+					JSON.stringify({
+						"secret-name-1": "secret_text",
+						"secret-name-2": "secret_text",
+					})
+				);
+
+				// User counter to pass different secrets to the request mock
+				const counter = 0;
+				setMockResponse(
+					`/accounts/:accountId/workers/scripts/:scriptName/secrets`,
+					"PUT",
+					([_url, accountId]) => {
+						expect(accountId).toEqual("some-account-id");
+						counter + 1;
```

Comment:
Same as above

Reply:
Check the inlinesnapshot in the assertion.

Reply:
L165 and L170 - it is always returning `Failed to create secret 0`.
This is the output that comes from the variable `counter`. The other numbers in the inline snapshot come from the json file. 
E.g. in `Error uploading secret for key: secret-name-1`, the key `secret-name-1` comes directly from the json file, and not from the response of the api request, so won't be affected by your variable `counter`.

To explain more clearly:
```js
const counter = 0;
console.log(counter); // -> 0

counter + 1; // doesn't update variable
console.log(counter); // -> 0

counter += 1; // Uncaught TypeError: Assignment to constant variable.
```

Reply:
Oh my bad, nice catch.